src/Network/Network/AzureFirewallPolicy/NewAzureFirewallPolicyCommand.cs

```diff
@@ -132,6 +132,12 @@ public class NewAzureFirewallPolicyCommand : AzureFirewallPolicyBaseCmdlet
         [ValidateNotNullOrEmpty]
         public PSManagedServiceIdentity Identity { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The private IP ranges to which traffic won't be SNAT'ed"
+        )]
+        public string[] PrivateRange { get; set; }
```

Comment:
I think the name of PrivateIpRange is more specific for this parameter, what's your thought?

Reply:
I think that makes sense. However, AzureFirewall already has this PrivateRange field defined for some time now. That's why AzureFirewallPolicy is naming it the same way to be consistent. Here is the design review for it:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/816

Reply:
Fine, got it.